api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -129,6 +136,31 @@ func (c *imageBuilder) GetMainAppPath(version *models.Version) (string, error) {
 	return baseImageTag.MainAppPath, nil
 }
 
+func (c *imageBuilder) getModelDependenciesUrl(ctx context.Context, version *models.Version) (string, error) {
+	artifactURL, err := c.gsutil.ParseURL(version.ArtifactURI)
+	if err != nil {
+		return "", err
+	}
+
+	condaEnvUrl := fmt.Sprintf("gs://%s/%s/model/conda.yaml", artifactURL.Bucket, artifactURL.Object)
+
+	condaEnv, err := c.gsutil.ReadFile(ctx, condaEnvUrl)
+	if err != nil {
+		return "", err
+	}
+
+	hash := sha256.New()
+	hash.Write([]byte(condaEnv))
+	hashEnv := hash.Sum(nil)
+
+	dependenciesUrl := fmt.Sprintf("gs://%s/merlin/dependencies/%x", artifactURL.Bucket, hashEnv)
```

Comment:
`merlin/dependencies` sounds like it's pip freeze, `modeldependencies`?

Reply:
updated to model_dependencies